dnscrypt-proxy/example-dnscrypt-proxy.toml

```diff
@@ -507,18 +507,18 @@ cache_neg_max_ttl = 600
 
 
 ######################################################
-#   Pattern-based allowlisting (blocklists bypass)   #
+#   Pattern-based allow lists (blocklists bypass)   #
 ######################################################
 
 ## Allowlists support the same patterns as blocklists
-## If a name matches a allowlist entry, the corresponding session
+## If a name matches an allowlist entry, the corresponding session
 ## will bypass names and IP filters.
 ##
 ## Time-based rules are also supported to make some websites only accessible at specific times of the day.
 
 [allowed_names]
 
-  ## Path to the file of allowlisting rules (absolute, or relative to the same directory as the config file)
+  ## Path to the file of allow list rules (absolute, or relative to the same directory as the config file)
```

Comment:
I commented on the original commit but I don't know how notifications work for that so I'll add a note here about "allowlisted" -> "allowed" a few lines below here.

Reply:
Sorry - it's possible I did an early push and obliterated it...  these commenting systems are good but not infallible.  Like me perhaps :-)  I've made this change too -- thanks for the careful and patient reviewing.

Reply:
No worries. :-)